contrib/devtools/check-translations.py

```diff
@@ -130,9 +130,23 @@ def check_all_translations():
                 # pick all numerusforms
                 if numerus:
                     translations = [i.text for i in translation_node.findall('numerusform')]
+                    if len(translations) == 0:
+                        print(f'Numerus message without numerusform translations, needs fix.')
```

Comment:
Could use some form of reference, to find what to actually fix. `<source>` contents? Same for line 141 and 147.